db/DB_Script.sql

```diff
@@ -1,47 +1,48 @@
-CREATE DATABASE teamC_Max;
+CREATE DATABASE teamC_Josh;
```

Comment:
Is this a change meant to be in the `main` branch, or was it a temporary change for local development?
If the latter, that implies the database name should be configurable.

Reply:
This was just a mistake in the script being corrected.